src/openrct2-ui/windows/PatrolArea.cpp

```diff
@@ -150,7 +150,9 @@ class PatrolAreaWindow final : public Window
     {
         auto mapTile = GetBestCoordsFromPos(screenCoords);
         if (!mapTile)
+        {
```

Comment:
Revert these unrelated changes for this PR.

Reply:
I accidentally created a brach from a strange place and can't revert
I'm going to make a new PR.